VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/command-centre/tests/command-centre.component-events.spec.ts

```diff
@@ -135,7 +135,7 @@ describe('CommandCentreComponent - Events', () => {
         component.hearings[1].getParticipants()[1].base.id = participant.id;
 
         participant.status = ParticipantStatus.Joining;
-        const message = new ParticipantStatusMessage(participant.id, participant.username, conferenceId, ParticipantStatus.Available);
+        const message = new ParticipantStatusMessage(participant.id, '', conferenceId, ParticipantStatus.Available);
```

Comment:
Could username be removed from `ParticipantStatusMessage` ?

Reply:
I think we have VIH-7083, it would be better to remove in separate PR for 7083